contract/src/test/kotlin/com/r3/corda/sdk/token/contracts/EvolvableTokenTests.kt

```diff
@@ -0,0 +1,719 @@
+package com.r3.corda.sdk.token.contracts
```

Comment:
Wow! These tests are very comprehensive - thanks!

Reply:
The complexity is caused by the need to split maintainers from participants.